checker/checkers.go

```diff
@@ -114,13 +116,25 @@ type typeAndPriority struct {
 }
 
 func determineTypeAndPriorityForPaths(assetPaths []string) (map[string]typeAndPriority, error) {
-	typeAndPriorities := make(map[string]typeAndPriority)
-	for _, assetPath := range assetPaths {
-		typeAndPriorityForAsset, err := determineTypeAndPriority(assetPath)
-		if err != nil {
-			return nil, err
-		}
-		typeAndPriorities[assetPath] = typeAndPriorityForAsset
+	typeAndPriorities := map[string]typeAndPriority{}
```

Comment:
nit: prefer to initialize map using `make`

```suggestion
	typeAndPriorities := make(map[string]typeAndPriority)
```

Reply:
Done